RELEASE.md

```diff
@@ -0,0 +1,42 @@
+# Zeebe Benchmark Helm Chart Release Process
+
+The charts are build, linted and tested on every push to the main branch. If the chart version
+(in `Chart.yaml`) changes a new github release with the corresponding packaged helm chart is
+created. The charts are hosted via github pages and use the release artifacts. We use the
+[chart-releaser-action](https://github.com/helm/chart-releaser-action) to release the charts.
+
+## Update Camunda Platform Helm chart
+
+Before the release make sure that the Chart version, in the [Chart.yaml](./charts/zeebe-benchmark/Chart.yaml) is up-to-date.
+
+This should be also done automatically via dependabot.
+
+## Process
+
+We are trying to automate as much as possible of the release process yet without sacrificing
+transparency so we are using PR release flow with minimal manual interactions.
+
+When it's time to release, just do the following steps.
+
+Locally, run:
+
+```
+make release.chores
+```
+
+This action will:
+
+- Locally pull latest updates to the `main` branch.
+- Locally create a new branch called `release` from `main` branch.
+- Bump chart version and make a commit.
+- Generate release notes and make a commit.
+- Push updated `release` branch to the repo.
+- Generate a link to open a PR with prefilled title and template.
+
+Next, all that you need to open the PR using the generated link and follow th checklist there.
```

Comment:
:upside_down_face: There's no checklist in the generated PR :D

Reply:
Added https://github.com/zeebe-io/benchmark-helm/blob/main/.github/PULL_REQUEST_TEMPLATE/release_template.md